CMakeLists.txt

```diff
@@ -248,10 +256,6 @@ else()
     set(GENERATED_SOURCE_DIR ${CMAKE_CURRENT_BINARY_DIR} CACHE STRING "Source directory where to find the generated source. Default value is CMAKE_CURRENT_BINARY_DIR")
 endif()
 
-if (${FEATURE_COMPILE_INLINE_MEMORY_ACCESSORS})
-    add_compile_definitions(USE_INLINE_MEMORY_ACCESSORS=1)
-endif()
-
```

Comment:
I cleaned a bit these compile time definitions to make them local to the VM library compilation and not global options to the entire project.